src/frontend/screens/Settings/ProjectSettings/SyncSettings.tsx

```diff
@@ -0,0 +1,146 @@
+    id: 'screens.SyncSettings.title',
+    defaultMessage: 'Sync Settings',
+  },
+  syncPreviews: {
+    id: 'screens.SyncSettings.syncPreviews',
+    defaultMessage: 'Sync Previews (Photos Only)',
+  },
+  syncPreviewsDescription: {
+    id: 'screens.SyncSettings.syncPreviewsDescription',
+    defaultMessage:
+      'Photos will sync at a reduced smaller size. Device will <bold>not</bold> sync audio or video.',
+  },
+  syncEverything: {
+    id: 'screens.SyncSettings.syncEverything',
+    defaultMessage: 'Sync Everything',
+  },
+  syncEverythingDescription: {
+    id: 'screens.SyncSettings.syncEverythingDescription',
+    defaultMessage:
+      'Your device will sync <bold>all</bold> content at full size, including photos, audio, and videos.<newline/><newline/>Note: This will use more storage.',
+  },
+  syncPreviewsBottomSheet: {
+    id: 'screens.SyncSettings.syncPreviewsButtonBottomSheet',
+    defaultMessage: 'Sync Previews?',
+  },
+  syncEverythingBottomSheet: {
+    id: 'screens.SyncSettings.syncEverythingButtonBottomSheet',
+    defaultMessage: 'Sync Everything?',
+  },
+  syncPreviewsDescriptionBottomSheet: {
+    id: 'screens.SyncSettings.syncPreviewsDescriptionBottomSheet',
+    defaultMessage:
+      'Your device will keep all existing data but new observations will sync in a smaller, preview size.<newline/><newline/>You will no longer sync Audio or Video.',
+  },
+  syncEverythingDescriptionBottomSheet: {
+    id: 'screens.SyncSettings.syncEverythingDescriptionBottomSheet',
+    defaultMessage:
+      'You are about to sync everything. This may increase the disk space used on your device.',
+  },
+  syncPreviewsBottomSheetConfirm: {
+    id: 'screens.SyncSettings.syncPreviewsBottomSheetConfirm',
+    defaultMessage: 'Sync Previews',
+  },
+});
+
+export const SyncSettings = () => {
+  const {formatMessage: t} = useIntl();
+  const syncSetting = usePersistedSettings(store => store.syncSetting);
+  const {setSyncSetting} = usePersistedSettingsAction();
+
+  const [modalType, setModalType] = React.useState<SyncSetting>(
+    () => syncSetting,
+  );
+
+  const {isOpen, openSheet, closeSheet, sheetRef} = useBottomSheetModal({
+    openOnMount: false,
+  });
+
+  const handleOptionChange = (value: SyncSetting) => {
+    setModalType(value);
+    openSheet();
+  };
+
+  const handleConfirm = () => {
+    setSyncSetting(modalType);
+    closeSheet();
+  };
+
+  const options: {
+    value: SyncSetting;
+    label: string;
+    hint?: string;
+  }[] = [
+    {
+      value: 'previews',
+      label: t(m.syncPreviews),
+      hint: t(m.syncPreviewsDescription),
+    },
+    {
+      value: 'everything',
+      label: t(m.syncEverything),
+      hint: t(m.syncEverythingDescription),
+    },
+  ];
+
+  return (
+    <ScrollView style={styles.container}>
+      <SelectOne
+        value={syncSetting}
+        onChange={handleOptionChange}
+        options={options}
+        radioButtonPosition="right"
+        color={SYNC_BACKGROUND}
```

Comment:
I used the color that was in the UI designs. Is that not the proper procedure?
<img width="647" alt="Screenshot 2024-09-05 at 2 52 25 PM" src="https://github.com/user-attachments/assets/e52d0e70-bd85-4674-a484-0f6ec283b29b">

Should we keep the component in case it is needed in the future?

Reply:
> I used the color that was in the UI designs. Is that not the proper procedure?

nah i think that's the correct approach. especially since it's a color that already exists in the codebase (albeit with a unnecessarily restrictive name).

> Should we keep the component in case it is needed in the future?

I don't think we should, but would leave that for a separate PR